TasteITServer/src/main/java/pl/jakubkonkol/tasteitserver/controller/PostController.java

```diff
@@ -98,4 +96,12 @@ public ResponseEntity<List<CommentDto>> getComments(@PathVariable String postId)
         List<CommentDto> commentDtos = commentService.getComments(postId);
         return ResponseEntity.ok(commentDtos);
     }
+
+    @GetMapping("/getPostsLikedByUser/{userId}")
```

Comment:
imo trochę dziwnie wygląda taki endpoint,(getPostsLikedByUser), może po prostu: /{userId}?

Reply:
Jak bedzie poprostu /{userId} to nie bedzie sie pierdolic z /{postId}?

Reply:
a no ta, ale nwm zmieniłbym moze na /likedby/{user}
bo w sensie po co robic "getposts..." skoro i tak zaczynamy od /posts w request mapping

Reply:
ta, likedBy/{userId} wydaje sie lepsze

Reply:
Trzeba sprawdzic wszystkie endpointy zeby nazewnictwo bylo spojne bo rzeczywiscie dawanie "get" w nazwie endpointow jesli i tak zaczynamy od np /posts, /user, itp i metoda jest GET, jest troche bez sensu